### PR TITLE
Feature/#574add possibilty to reset end/start time

### PR DESCRIPTION
**Pull request by DenysIvashchenko**
Added possibility to reset start/end Time for workshop-filtering. Added possibility to add only endTime
clearBtn

### CODE REVIEW

File: src/app/shared/components/filters-list/working-hours/working-hours.component.scss [hunk 1]
```diff
@@ -74,4 +74,28 @@
 ::ng-deep .timepicker__main-content{
```
Comment (dmitryy90) [inline src/app/shared/components/filters-list/working-hours/working-hours.component.scss:74]: please isolate ng-deep

File: src/app/shared/components/filters-list/working-hours/working-hours.component.ts [hunk 2]
```diff
@@ -55,18 +55,30 @@ export class WorkingHoursComponent implements OnInit, OnDestroy {
       takeUntil(this.destroy$),
       debounceTime(300),
       distinctUntilChanged(),
-    ).subscribe((time: string) => this.store.dispatch(new SetStartTime(time.split(':')[0])));
+    ).subscribe((time: string) => this.store.dispatch(new SetStartTime(time?.split(':')[0])));
```
Comment (dmitryy90) [inline src/app/shared/components/filters-list/working-hours/working-hours.component.ts:58]: what the reason of this change?

File: src/app/shared/components/filters-list/working-hours/working-hours.component.html [hunk 3]
```diff
@@ -6,14 +6,16 @@
       {{day.value}}</div>
   </div>
   <p class="filter-text">Години роботи</p>
-  <mat-form-field>
+  <mat-form-field> 
     <div class="wrapper" fxLayout='row' fxLayoutAlign='start' fxLayoutGap='1rem'>
       <p class="filter-text">Від</p>
       <input class="time-input" type="text" matInput [ngxMatTimepicker]="timepickerStart" format="24"
-        [formControl]="startTimeFormControl">
+        [formControl]="startTimeFormControl" [max]="getMaxTime()">
```
Comment (dmitryy90) [inline src/app/shared/components/filters-list/working-hours/working-hours.component.html:13]: to use functions in data binding is bad approach and can cause poor performance, please use variable instead

**Review by dmitryy90 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/filters-list/working-hours/working-hours.component.ts (hunk 2 above)
Comment (DenysIvashchenko) [reply src/app/shared/components/filters-list/working-hours/working-hours.component.ts:58]: because of new functionality added for the reset input

**Review by dmitryy90 — APPROVED** [review]

**Pull request merged**